sdk/enter.go

```diff
@@ -188,26 +188,30 @@ func (e *enter) MountGnupg() error {
 		origHome = filepath.Join(e.User.HomeDir, ".gnupg")
 	}
 
-	if _, err := os.Stat(origHome); err != nil {
-		// Skip but do not pass along $GNUPGHOME
-		return os.Unsetenv("GNUPGHOME")
+	// gpg misbehaves in the sdk with GNUPGHOME set to anything but ~/.gnupg
+	// so always unset it
+	if err := os.Unsetenv("GNUPGHOME"); err != nil {
```

Comment:
Does this remain unset after cork exits?

Reply:
Tested and nope. Only for this process (and things it launches?)